### PR TITLE
prepare single coordinator read test

### PR DESCRIPTION
**Pull request by jnels124**
prepare single coordinator read test

### CODE REVIEW

**Comment by jnels124** [comment]
Want to do a quick k6 test to see if it is worth the effort of keeping the replica

**Review by steven-sheehy — APPROVED** [review]

**Review by jascks — DISMISSED** [review]
LGTM

File: clusters/mainnet-staging-na/mainnet/helmrelease.yaml [hunk 1]
```diff
@@ -88,6 +88,9 @@ spec:
     rosetta:
       env:
         HEDERA_MIRROR_ROSETTA_NETWORK: "mainnet"
+    stackgres:
```
Comment (xin-hedera) [inline clusters/mainnet-staging-na/mainnet/helmrelease.yaml:91]: shouldn't it go to mainnet-citus namespace?

**Review by xin-hedera — CHANGES_REQUESTED** [review]

**Review by steven-sheehy — APPROVED** [review]

**Review by xin-hedera — APPROVED** [review]
LGTM

**Pull request merged**